### PR TITLE
Feature/add currency in merchandise

### PR DESCRIPTION
**Pull request by HLTVProxy**
- add coin merchandise currency display
- also need to watch other repos which are app, element and backend

### CODE REVIEW

**Comment by HLTVProxy** [comment]
[(3) 【商品】商品希望也能用代幣購買 [1]](https://trello.com/c/IN2yZDyB/3475-3-%E3%80%90%E5%95%86%E5%93%81%E3%80%91%E5%95%86%E5%93%81%E5%B8%8C%E6%9C%9B%E4%B9%9F%E8%83%BD%E7%94%A8%E4%BB%A3%E5%B9%A3%E8%B3%BC%E8%B2%B7-1)

**Comment by futianshen** [comment]
@HLTVProxy 
There are too many unrelated commits on this branch, you need to rebase this branch on develop branch to solve conflicts first

File: src/components/sale/SaleCollectionAdminCard.tsx [hunk 1]
```diff
@@ -364,7 +372,7 @@ const SaleCollectionAdminCard: React.VFC<{
               <div className="col-3">{currencyFormatter(shipping.fee || 0)}</div>
             </div>
           )}
-
+          {console.log(orderDiscounts)}
```
Comment (futianshen) [inline src/components/sale/SaleCollectionAdminCard.tsx:375]: `console.log` should be removed

Comment (HLTVProxy) [reply src/components/sale/SaleCollectionAdminCard.tsx]: had removed console.log in below commit

**Review by torai55 — APPROVED** [review]

**Pull request merged**